src/common/config/index.js

```diff
@@ -324,6 +327,34 @@ export default class Config extends EventEmitter {
     return newTeams;
   }
 
+  /**
+   * Apply a default sort order to the team list, if no order is specified.
+   * @param {array} teams to sort
+   */
+  sortUnorderedTeams(teams) {
+    // Make a best pass at interpreting sort order. If an order is not specified, assume it is 0.
+    //
+    const newTeams = teams.sort((x, y) => {
+      if (x.order == null) {
+        x.order = 0;
+      }
+      if (y.order == null) {
+        y.order = 0;
+      }
+      return (x.order === y.order) ? 0 : ((x.order > y.order) ? 1 : -1); // eslint-disable-line no-nested-ternary
+    });
+
+    // Now re-number all items from 0 to (max), ensuring user's sort order is preserved. The
+    // new tabbed interface requires an item with order:0 in order to raise the first tab.
+    //
+    let i = 0;
+    newTeams.forEach((team) => {
+      team.order = i++;
+    });
```

Comment:
We can remove handling the `i` by using its actual index
```suggestion
    newTeams.forEach((team, i) => {
      team.order = i;
    });
```